pkg/pkg.ml

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env ocaml
+#use "topfind"
+#require "topkg"
+open Topkg
+
+
+let () =
+  Pkg.describe "tuntap" @@ fun c ->
+  Ok [
+    Pkg.mllib "lib/tuntap.mllib";
+    Pkg.clib "lib/libtuntap_stubs.clib";
+    Pkg.test "test/getifaddrs_test";
```

Comment:
@gjaldon I don't know how to push that commit here... but I opened #20, which is your commit and mine.

`.a` files are standalone archives, where `.o` files are the object files - this corresponds to `.cmxa` vs `.o` in OCaml (read more about it [here](http://stackoverflow.com/questions/25422560/what-are-all-these-cma-z-files-and-when-we-need-them#25423837) and [here](https://en.wikipedia.org/wiki/Static_library)).